packages/wrangler/e2e/miniflare-remote-resources.test.ts

```diff
@@ -427,28 +427,29 @@ describe.each([...testCases, mtlsTest])("Mixed Mode for $name", (testCase) => {
 async function runTestCase<T>(
 	testCase: TestCase<T>,
 	helper: WranglerE2ETestHelper,
-	{ disableMixedMode } = { disableMixedMode: false }
+	{ disableRemoteBindings } = { disableRemoteBindings: false }
 ) {
-	const { experimental_startMixedModeSession } = await helper.importWrangler();
+	const { experimental_startRemoteProxySession } =
+		await helper.importWrangler();
 	const { Miniflare } = await helper.importMiniflare();
 	await helper.seed(path.resolve(__dirname, "./seed-files/mixed-mode-workers"));
```

Comment:
again `mixed-mode-wortkers` (although I notice that you did rename the directory, so probably these tests should fail right now)